2024-02-13 12:20:00,632 INFO    MainThread:105948 [wandb_setup.py:_flush():76] Current SDK version is 0.16.3
2024-02-13 12:20:00,632 INFO    MainThread:105948 [wandb_setup.py:_flush():76] Configure stats pid to 105948
2024-02-13 12:20:00,632 INFO    MainThread:105948 [wandb_setup.py:_flush():76] Loading settings from /home-nfs/fx2024/.config/wandb/settings
2024-02-13 12:20:00,633 INFO    MainThread:105948 [wandb_setup.py:_flush():76] Loading settings from /home-nfs/fx2024/NLP/NLP/wandb/settings
2024-02-13 12:20:00,633 INFO    MainThread:105948 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-02-13 12:20:00,633 INFO    MainThread:105948 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-02-13 12:20:00,633 INFO    MainThread:105948 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'main.py', 'program_abspath': '/home-nfs/fx2024/NLP/NLP/main.py', 'program': '/home-nfs/fx2024/NLP/NLP/main.py'}
2024-02-13 12:20:00,633 INFO    MainThread:105948 [wandb_init.py:_log_setup():526] Logging user logs to /home-nfs/fx2024/NLP/NLP/wandb/run-20240213_122000--bs-32-e-6-lr-1e-04-dp-1e-01/logs/debug.log
2024-02-13 12:20:00,633 INFO    MainThread:105948 [wandb_init.py:_log_setup():527] Logging internal logs to /home-nfs/fx2024/NLP/NLP/wandb/run-20240213_122000--bs-32-e-6-lr-1e-04-dp-1e-01/logs/debug-internal.log
2024-02-13 12:20:00,633 INFO    MainThread:105948 [wandb_init.py:init():566] calling init triggers
2024-02-13 12:20:00,634 INFO    MainThread:105948 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {'learning_rate': 0.0001, 'architecture': 'GPT2', 'dataset': 'wikitext', 'epochs': 6}
2024-02-13 12:20:00,634 INFO    MainThread:105948 [wandb_init.py:init():616] starting backend
2024-02-13 12:20:00,634 INFO    MainThread:105948 [wandb_init.py:init():620] setting up manager
2024-02-13 12:20:00,637 INFO    MainThread:105948 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-02-13 12:20:00,641 INFO    MainThread:105948 [wandb_init.py:init():628] backend started and connected
2024-02-13 12:20:00,652 INFO    MainThread:105948 [wandb_init.py:init():720] updated telemetry
2024-02-13 12:20:00,667 INFO    MainThread:105948 [wandb_init.py:init():753] communicating run to backend with 90.0 second timeout
2024-02-13 12:21:30,698 ERROR   MainThread:105948 [wandb_init.py:init():779] encountered error: Run initialization has timed out after 90.0 sec. 
Please refer to the documentation for additional information: https://docs.wandb.ai/guides/track/tracking-faq#initstarterror-error-communicating-with-wandb-process-
2024-02-13 12:21:30,829 ERROR   MainThread:105948 [wandb_init.py:init():1194] Run initialization has timed out after 90.0 sec. 
Please refer to the documentation for additional information: https://docs.wandb.ai/guides/track/tracking-faq#initstarterror-error-communicating-with-wandb-process-
Traceback (most recent call last):
  File "/home-nfs/fx2024/mc3/envs/fei-venv/lib/python3.11/site-packages/wandb/sdk/wandb_init.py", line 1176, in init
    run = wi.init()
          ^^^^^^^^^
  File "/home-nfs/fx2024/mc3/envs/fei-venv/lib/python3.11/site-packages/wandb/sdk/wandb_init.py", line 785, in init
    raise error
wandb.errors.CommError: Run initialization has timed out after 90.0 sec. 
Please refer to the documentation for additional information: https://docs.wandb.ai/guides/track/tracking-faq#initstarterror-error-communicating-with-wandb-process-
2024-02-13 12:21:31,838 WARNING MsgRouterThr:105948 [router.py:message_loop():77] message_loop has been closed
